### PR TITLE
chore: Use pixi exec for autoupdating lockfiles

### PR DESCRIPTION
**Pull request by pavelzw**

### CODE REVIEW

File: docs/advanced/updates_github_actions.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Update lockfiles
         run: |
           set -o pipefail
-          pixi update --json | pixi exec pixi-diff-to-markdown >> diff.md
+          pixi update --json --no-install | pixi exec pixi-diff-to-markdown >> diff.md
```
Comment (Hofer-Julian) [inline docs/advanced/updates_github_actions.md:45]: We removed it on purpose from the docs. It doesn't work if the manifest has pypi-dependencies

Comment (pavelzw) [reply docs/advanced/updates_github_actions.md:45]: ah okay, didn't notice that

File: .github/workflows/update-lockfiles.yml [hunk 2]
```diff
@@ -21,8 +21,7 @@ jobs:
       - name: Update lockfiles
         run: |
           set -euo pipefail
-          pixi global install pixi-diff-to-markdown
-          pixi update --json --no-install | pixi-diff-to-markdown >> diff.md
+          pixi update --json --no-install | pixi exec pixi-diff-to-markdown >> diff.md
```
Comment (Hofer-Julian) [inline .github/workflows/update-lockfiles.yml:24]: This part still makes sense though :)

File: docs/advanced/updates_github_actions.md (hunk 1 above)
Comment (Hofer-Julian) [reply docs/advanced/updates_github_actions.md:45]: Yeah, I should have tagged you

**Review by Hofer-Julian — APPROVED** [review]
Thanks!

**Pull request merged**